robocop/checkers/spacing.py

```diff
@@ -472,3 +473,29 @@ def get_indent(node):
                 break
             indent_len += len(token.value.expandtabs(4))
         return indent_len
+
+
+class LeftAlignedChecker(VisitorChecker):
+    """ Checker for left align. """
+    rules = {
+        "1014": (
+            "variable-should-left-aligned",
+            "Variable in variable section should be left aligned.",
+            RuleSeverity.ERROR
+        )
+    }
+
+    def __init__(self):
+        super().__init__()
+        self.disabled = not IS_RF4
```

Comment:
Great that you've discovered this 'feature' - I think it's not covered in any contribution guides (and I definietly need to describe it)

Reply:
In RF 3.2, the error is handled by 0401 parsing-error rule.